scipy/_lib/_util.py

```diff
@@ -815,15 +814,19 @@ def _rng_spawn(rng, n_children):
     return child_rngs
 
 
-def _get_nan(*data):
+def _get_nan(*data, xp=None):
+    if data == ():
```

Comment:
When is this branch useful?

Reply:
TBH only an input validation test I came across. It allowed the right error message to be raised. I can look into adjusting the function that was being tested.

Reply:
Removed it.